PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,13 +2,13 @@
 
 - try to explain what the code is about
 - try to follow [PEP0008](https://www.python.org/dev/peps/pep-0008/)
-- make the pull request against the ["v0.6" branch](https://github.com/Bitmessage/PyBitmessage/tree/v0.6)
```

Comment:
Please don't remove this line regarding branches, only the one regarding fast-forward merging